charts/tidb-operator/templates/controller-manager-rbac.yaml

```diff
@@ -117,7 +116,59 @@ roleRef:
   kind: ClusterRole
   name: {{ .Release.Name }}:tidb-controller-manager
   apiGroup: rbac.authorization.k8s.io
-{{- if (not .Values.clusterScoped) }}
+{{- else }}
+{{- if and .Values.clusterPermissions (or .Values.clusterPermissions.nodes .Values.clusterPermissions.persistentvolumes .Values.clusterPermissions.storageclasses) }}
```

Comment:
We have to cover the following cases:
1. No such configurations in the values.yaml, e.g. with values.yaml of the previous version
    We have to set the missed configurations to `true`, refer to the logic for  `.Values.controllerManager` in L4.
2. With those configurations and set to `true`
3. With those configurations and set to `false`